packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/union/models/ResourceOperationStatusOperationStatusResultError.java

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Caused by [case2](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#2-lro-initial-response-has-public-access-in-tcgc)